xen.spec.in

```diff
@@ -556,8 +468,6 @@ autoreconf
     --libexecdir=%{_libexecdir} \
     --with-system-seabios=%{seabiosloc} \
     --with-system-ipxe=/usr/share/ipxe/8086100e.rom \
-    --enable-vtpm-stubdom \
-    --enable-vtpmmgr-stubdom \
```

Comment:
I think there is also `--disable-stubdom` (see main configure help).

And you might also need to adjust %files section in this spec.

Reply:
It's already handled. I enforced ```%define build_stubdom 0```.